.mage/js.go

```diff
@@ -146,7 +144,7 @@ func (js Js) DevDeps() error {
 // Deps installs the javascript dependencies.
 func (js Js) Deps() error {
 	files, readErr := ioutil.ReadDir("node_modules")
-	changed, targetErr := target.Path("node_modules", "./package.json", "./yarn.lock")
+	changed, targetErr := target.Path("node_modules", "./package.json", "./yarn.lock", "./sdk/js/dist")
```

Comment:
Please revert this change, this will be handled in https://github.com/TheThingsNetwork/lorawan-stack/pull/2751 - perhaps, you can suggest to apply this there?

Reply:
Unfortunately forgot to suggest this change there. I'll keep it in this PR.